hstest/stage/__init__.py

```diff
@@ -2,8 +2,10 @@
     'StageTest',
     'DjangoTest',
     'FlaskTest',
+    'PlottingTest'
```

Comment:
There should be a comma also. Reason: git diff will be cleaner the next time we add another class to the list (1 line instead of 2 lines - comma on this line + actual class on the next line.)